### DISCUSSION TITLE
New Source: RabbitMQ

### DISCUSSION THREAD

**Discussion by amitza** [airbytehq/airbyte#36083] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Source: [RabbitMQ](https://www.rabbitmq.com)
Destination: S3, RabbitMQ 
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
Support AMQP 0-9-1

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
Will be used to ingest json documents data to batched parquet files in our S3
* How often do you want to run syncs?
The best option would be to wait for a batch of messages (up to a certain number or size of message)
* If this is an API source connector, which entities/endpoints do you need supported?
A message with a single Json or with a list of Jsons

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Probably [Apache NiFi](https://nifi.apache.org/)

## Are you willing to submit a PR?
I would love to try and submit a PR!

**Comment by marcosmarxm** [comment]
Awesome @amitza let me know if you need any assistance here.

**Comment by amitza** [comment]
> Awesome @amitza let me know if you need any assistance here.

Thanks! I've started working on a [PR](https://github.com/amitza/airbyte/pull/1) and I didn't really understand what's the `discover` method is for

**Comment by marcosmarxm** [comment]
Check here: https://docs.airbyte.com/understanding-airbyte/airbyte-protocol#discover

**Comment by agapebondservant** [comment]
Hi, also interested in getting an OOTB Rabbit source connector, is the `discover` method the only approach that will handle this?

**Comment by weistrand** [comment]
I'm interested in getting data from AMQP into Airbyte. Looks like no one has worked on this for a long time? Should I give it a try?
